### ISSUE TITLE
tardis/data/tardis_example_config.yml is not a valid config file

### ISSUE THREAD

**Issue by lukeshingles** [tardis-sn/tardis#521]
Just to get tardis started on something, I tried running 'tardis tardis_example_config.yml output_spectrum.dat' in the folder 'tardis/data', which resulted in the yaml parsing errors.

It looks like this file is an old version of tardis_example.yml from https://www.dropbox.com/s/svvyr5i7m8ouzdt/tardis_example.tar.gz

So a couple of possible solutions:
- remove the file from the repo (in case anyone other than me is confused by its presence)
- update the file to match the tardis_example.tar.gz version (with the small atomic database)
- add the tardis example files to the repository (using git LFS for the 74MB atomic database)

**Comment by lukeshingles** [comment]
It looks like atom_data.h5 isn't a valid atomic database for the current version either, so option 2 doesn't work easily.

**Comment by karandesai-96** [comment]
My opinion is of going with the third option. It will do a two way benefit:
- A new user need not open the Dropbox link mentioned in the documentation and download the file separately to run the tardis_example.
- The **kurucz_cd23_chianti_H_He.h5** atom data file is also used in W7 and Talytha's runs - the ones I have encountered as slow tests, and I think it is a standard setup useful in many subsequent runs I encounter. I had to download it separately and I kept it within the directory structure of Tardis. Unfortunately the next time I ran `git clean -dfx`, it was lost and I had to download it again. Having it in the directory structure would ease it all up - not specifically aligned to my development work, but also after the Test suite gets ready.

I feel that even on the server, doing a **wget** and downloading a 25 MB .tar.gz file for the atom data file for every build doesn't sound good.

Also, as it's at our disposal, we can leverage the power of git LFS to our needs.

Thoughts ?

**Comment by wkerzendorf** [comment]
@lukeshingles A couple of points. TARDIS or other python packages should not be run from within their structure. the `data` directory should only be used for TARDIS internal data. 

I think for now we leave them as is (as the documentation is not pointing in this direction), but we'll leave this issue open to remind us to clean this in the future. 

does this sound like a plan @lukeshingles ?

**Comment by unoebauer** [comment]
@wkerzendorf independently of whether or not one should run tardis within its own structure, I think @lukeshingles has a point. Doesn't make much sense to keep stuff around which is deprecated and/or incompatible with the current code.

**Comment by unoebauer** [comment]
With the current config system, this example file still crashes 
```
ScannerError: while scanning a simple key
  in "tardis_example_config.yml", line 39, column 9
could not found expected ':'
  in "tardis_example_config.yml", line 40, column 9
```

Still some work to do :)

**Comment by wkerzendorf** [comment]
The example files should be deleted in the data directory

**Comment by ftsamis** [comment]
@wkerzendorf I was thinking the same, but on the other hand, I prefer having the example yml file version controlled, rather than in an arbitrary zip file somewhere. 

Would it be a good idea now that we have https://github.com/tardis-sn/tardis-atomdata to change the documentation a bit so it directs users to fetch the yml from tardis-sn/tardis branch and the atom data from the tardis-sn/tardis-atomdata branch? If not, the example files should be indeed removed from the branch.

**Comment by wkerzendorf** [comment]
@ftsamis - the example file will likely go away and unseres will only download the atomic data file and look at examples in the documentation

**Comment by unoebauer** [comment]
Let's clean up the tardis working tree for 2.0

**Comment by unoebauer** [comment]
Resolved by #797.
